actors/miner/tests/miner_actor_test_wpost.rs

```diff
@@ -0,0 +1,123 @@
+use fil_actor_miner as miner;
+
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::sector::RegisteredSealProof;
+
+use fvm_ipld_bitfield::{BitField, UnvalidatedBitField};
+
+mod util;
+
+const DEFAULT_SECTOR_EXPIRATION: u64 = 220;
+
+#[test]
+fn basic_post_and_dispute() {
+    let period_offset = ChainEpoch::from(100);
+    let precommit_epoch = ChainEpoch::from(1);
+
+    let mut h = util::ActorHarness::new(period_offset);
```

Comment:
I suggest `use util::*` for the accompanying crate.

Reply:
yeah, its ok for util. I generally avoid blanket imports, but here it makes sense.

Reply:
done, less noise now.